<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>559</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Toney</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,0,0">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Value Position:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="valPos">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Amplitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volumeSlider">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="numDigits">
         <number>3</number>
        </property>
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="ClusterBox" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>792</width>
          <height>76</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="Pool" name="poolWidget" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Open_Directory"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add_cluster"/>
    <addaction name="action_Reclassify"/>
    <addaction name="action_F0_Params"/>
    <addaction name="action_Value_Position"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Show_Frame_ID"/>
    <addaction name="action_Show_Speaker_ID"/>
    <addaction name="action_Resize_Column_Width"/>
    <addaction name="action_Playback_Delay"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Add_cluster">
   <property name="text">
    <string>&amp;Add Cluster</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Show_Frame_ID">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Frame ID</string>
   </property>
  </action>
  <action name="action_Show_Speaker_ID">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Speaker ID</string>
   </property>
  </action>
  <action name="action_Playback_Delay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Playback &amp;Delay</string>
   </property>
  </action>
  <action name="action_Open_Directory">
   <property name="text">
    <string>Open &amp;Directory</string>
   </property>
  </action>
  <action name="action_Resize_Column_Width">
   <property name="text">
    <string>Fit Column &amp;Width</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>User Documentation</string>
   </property>
  </action>
  <action name="action_Reclassify">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reclassify</string>
   </property>
  </action>
  <action name="action_F0_Params">
   <property name="text">
    <string>F0 Parameters</string>
   </property>
  </action>
  <action name="action_Value_Position">
   <property name="text">
    <string>Value Position</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClusterBox</class>
   <extends>QWidget</extends>
   <header>clusterbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pool</class>
   <extends>QWidget</extends>
   <header location="global">pool.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>volumeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
